vm-presets.md

```diff
@@ -0,0 +1,256 @@
+Virtual Machine Presets
+=============================
+
+`VirtualMachinePresets` are an extension to general `VirtualMachine`
+configuration behaving much like `PodPresets` from Kubernetes. When a
+`VirtualMachine` is created, any applicable `VirtualMachinePresets`
+will be applied to the existing spec for the `VirtualMachine`. This allows
+for re-use of common settings that should apply to multiple `VirtualMachines`.
+
+
+Usage
+------------------------
+
+KubeVirt uses Kubernetes `Labels` and `Selectors` to determine which
+`VirtualMachinePresets` apply to any given `VirtualMachine`, similarly to how
```

Comment:
@stu-gott I think that you now implemented it differently (you also partially apply the preset).